source/write/bulk-write.txt

```diff
@@ -0,0 +1,343 @@
+--------------------------
+
+After you define a model instance for each operation you want to perform,
+pass a list of these instances to the ``bulkWrite()`` method.
+By default, the method runs the operations in the order
+specified by the list of models.
+
+The following example performs multiple write operations by using the
+``bulkWrite()`` method:
+
+.. io-code-block::
+
+   .. input:: /includes/write/bulk-write.scala
+      :start-after: start-bulk-write-mixed
+      :end-before: end-bulk-write-mixed
+      :language: scala
+      :dedent:
+
+   .. output::
+      
+      Success: AcknowledgedBulkWriteResult{insertedCount=1, matchedCount=1, removedCount=1, modifiedCount=1, upserts=[], inserts=[BulkWriteInsert{index=0, id=BsonObjectId{value=...}}]}
+      Completed
+
+If any of the write operations fail, the {+driver-short+} raises a
+``BulkWriteError`` and does not perform any further operations.
+``BulkWriteError`` provides a ``details`` item that includes the
+operation that failed, and details about the exception.
+
+.. note::
+
+   When the driver runs a bulk operation, it uses the write concern of the
+   target collection. The driver reports all write concern errors after
+   attempting all operations, regardless of execution order.
+
+Customize Bulk Write Operation
+------------------------------
+
+The ``bulkWrite()`` method optionally accepts a parameter which
+specifies options you can use to configure the bulk write
+operation. If you don't specify any options, the driver performs the
+bulk operation with default settings.
+
+The following table describes the setter methods that you can use to
+configure a ``BulkWriteOptions`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``ordered()``
+     - | If ``true``, the driver performs the write operations in the order
+         provided. If an error occurs, the remaining operations are not
+         attempted. 
+       |
+       | If ``false``, the driver performs the operations in an
+         arbitrary order and attempts to perform all operations.
+       | Defaults to ``true``.
+
+   * - ``bypassDocumentValidation()``
+     - | Specifies whether the update operation bypasses document validation. This lets you 
+         update documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+       | Defaults to ``false``.
+
+   * - ``comment()``
+     - | Sets a comment to attach to the operation.
+
+   * - ``let()``
+     - | Provides a map of parameter names and values to set top-level
+         variables for the operation. Values must be constant or closed
+         expressions that don't reference document fields.
+
+The following code creates options and uses the ``ordered(false)`` option to
```

Comment:
S: I think this wording makes slightly more sense, since `ordered(false)` isn't the option:
```suggestion
The following code creates options and sets the ``ordered`` option to ``false`` to
```